### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**Please provide a description of this PR:**

### CODE REVIEW

**Review by ramaraochavali — APPROVED** [review]

File: pilot/pkg/serviceregistry/kube/controller/ambient/workloads.go [hunk 1]
```diff
@@ -50,58 +50,58 @@ import (
...
-			AuthorizationPolicies,
-			PeerAuths,
-			Waypoints,
-			WorkloadServices,
+			meshConfig,
+			authorizationPolicies,
+			peerAuths,
+			waypoints,
+			workloadServices,
 			WorkloadServicesNamespaceIndex,
```
Comment (howardjohn) [inline pilot/pkg/serviceregistry/kube/controller/ambient/workloads.go:76]: If we are going to change the casing we should do it to these as well

**Comment by hzxuzhonghu** [comment]
>👎 to this change, the casing was intentionally done to provide a clear distinction these were Collections which CANNOT be access directly. With this change it is really easy to accidentally mix up a local list with a Collection

I donot think there is a sepc about using upper cased args of a function

**Comment by hzxuzhonghu** [comment]
If it is toprevent  mixup, some more functions face the same issues

**Comment by howardjohn** [comment]
I think all the krt functions are using this casing for Collection?

**Comment by hzxuzhonghu** [comment]
krt is not a reason to violates golang spec

**Pull request merged**